Backend/app/common/PropertiesManager.py

```diff
@@ -79,8 +79,9 @@ def _set_attributes(self, config_section: str):
 
         """
         for key, value in self.config.items(config_section):
-            if value == "":
+            if value == "" or None:
```

Comment:
I think last comment might have confused you. In Python you have to either `if value=="" or value is None` instead. You can use also `if value in ["",None]`

Reply:
@AntonioMrtz , sorry for that i knew this still made an error.